dll/win32/msi/handle.c

```diff
@@ -246,8 +246,8 @@ int msiobj_release( MSIOBJECTHDR *info )
     {
         if( info->destructor )
             info->destructor( info );
-        msi_free( info );
         TRACE("object %p destroyed\n", info);
+		msi_free( info );
```

Comment:
Wrong/Tab indentation!
Then, maybe update the trace message too? Or rather use a copy of the pointer?